lamindb/models/history.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class History(models.Model):
+    """Stores the edit history for all LaminDB tables."""
+
+    seqno = models.AutoField(primary_key=True)
+    migration_history_id = models.BigIntegerField()
+    id = models.UUIDField()
+    table_name = models.CharField(max_length=100)
+    record_id = models.BigIntegerField()
```

Comment:
I think we need to use the record uid here because the integer record ids will be meaningless across two copies of the database. See https://github.com/laminlabs/lamindb/pull/2642#discussion_r2038234500